docs/osr-resources/playbook.md

```diff
@@ -0,0 +1,89 @@
+---
+id: osr-playbook
+title: OSR Playbook
```

Comment:
The title "OSR Playbook" doesn't reflect the contents of the document, which are about common problems when starting an OSPO. Please change.